packages/react-native/src/index.ts

```diff
@@ -118,12 +122,6 @@ class Honeybadger extends Client {
     const notice = {
       name: `React Native Android ${data.type}`,
       message: data.message || '',
-      details: {
```

Comment:
This was copied from the original package, but I just removed the `details` here because I took a look at [the java code that emits the event](https://github.com/honeybadger-io/honeybadger-js/blob/f0d361720144297d20dc7d08712f0ec5208aa564/packages/react-native/android/src/main/java/io/honeybadger/HoneybadgerReactNativeModule.java#L76C28-L76C28) and I don't think any of these fields will ever be populated.